storefront.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  defaultLocale: "en-US",
+  locales: ["fr-fr", "pl-pl"],
```

Comment:
`locales` should contain a list of all possible languages including the default one.

Reply:
Storefront config file will be submitted as separate PR.